zap/src/main/java/org/zaproxy/zap/extension/script/ExtensionScript.java

```diff
@@ -1039,6 +1039,8 @@ private int addScriptsFromDir(File dir, ScriptType type, String targetEngineName
                                             type,
                                             false,
                                             f);
+                            if (this.getScriptParam().isEnableScriptsFromDirs())
+                              this.setEnabled(sw,true);
```

Comment:
Should have braces and proper formatting

Reply:
I thought one expression conditionals are preferred without braces, will fix now. Where can I see the formatting guidelines for ZAP? The formatting that jdtls applies completely changes the indentation and other aspects of the file, so I don't think i can follow its standard.

Reply:
https://www.zaproxy.org/docs/developer/verifying-your-changes/#code-formatting 😁

Reply:
The braces thing isn’t explicit, you can likely find out laces in the code base where they aren’t used on single statements. However, we’ve been moving to use them more consistently.

For formatting there are gradle tasks to apply and check.

Reply:
Neat. Seems like spotlessCheck doesn't complain anymore.